deployments/goerli/solcInputs/3a32f099b226e81e32ffbca8f8873f42.json

```diff
@@ -235,8 +235,8 @@
     "contracts/testHelpers/PayableContract.sol": {
       "content": "// SPDX-License-Identifier: MIT\npragma solidity ^0.8.4;\n\n// A contract that accepts ETH\ncontract PayableContract {\n    fallback() external payable {}\n}\n"
     },
-    "contracts/testHelpers/RevocationTestHelper.sol": {
-      "content": "// SPDX-License-Identifier: MIT\npragma solidity ^0.8.4;\n\nimport \"@openzeppelin/contracts/utils/introspection/ERC165.sol\";\n\nimport \"../tagsAuthPolicies/ITagsAuthPolicy.sol\";\n\ncontract RevocationTestHelper is ITagsAuthPolicy, ERC165 {\n    bytes32 private _onTagClaimedRetVal;\n    bool private _tagCanBeRevokedRetVal = true;\n\n    function supportsInterface(bytes4 interfaceID) public view virtual override(IERC165, ERC165) returns (bool) {\n        return interfaceID == type(ITagsAuthPolicy).interfaceId || super.supportsInterface(interfaceID);\n    }\n\n    function canClaimTag(\n        bytes32,\n        string calldata,\n        address,\n        address,\n        bytes calldata\n    ) external view virtual override returns (bool) {\n        return true;\n    }\n\n    function onTagClaimed(\n        bytes32,\n        string calldata,\n        address,\n        address,\n        bytes calldata\n    ) external virtual override returns (bytes32 tagToRevoke) {\n        return _onTagClaimedRetVal;\n    }\n\n    function tagCanBeRevoked(\n        address,\n        bytes32,\n        string calldata,\n        bytes calldata\n    ) external view virtual override returns (bool) {\n        return _tagCanBeRevokedRetVal;\n    }\n\n    // solhint-disable func-name-mixedcase\n    function stub_tagCanBeRevokedReturnVal(bool retVal) external {\n        _tagCanBeRevokedRetVal = retVal;\n    }\n\n    function stub_onTagClaimedReturnVal(bytes32 retVal) external {\n        _onTagClaimedRetVal = retVal;\n    }\n    // solhint-enable func-name-mixedcase\n}\n"
+    "contracts/testHelpers/TagTestHelper.sol": {
```

Comment:
Inconsequential in this case, but I'd make sure not to commit edits to the `deployments` directory unless we've made a new on-chain deployment somewhere